lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -92,6 +100,10 @@ def parsed_step_navs
         content_item.dig("links", "part_of_step_navs").to_a
       end
 
+      def parsed_related_to_step_navs
+        content_item.dig("links", "related_to_step_navs").to_a
```

Comment:
`related_to_step_navs` is an array. You shouldn't need to convert it to an array.

Reply:
I copied the function above (line 99-101), should I update the above function too?

Reply:
Yes, to keep it consistent. Thanks.

Reply:
Changing this causes a lot of tests to fail...